moveit_core/planning_scene/src/planning_scene.cpp

```diff
@@ -48,6 +48,8 @@
 #include <memory>
 #include <set>
 
+rclcpp::Logger logger = rclcpp::get_logger("planning_scene");
```

Comment:
I see this done differently in almost every PR so far. Let's keep this standardized.

Reply:
I think that we are messing this up a little bit, lets just specify a name and use the same name on the entire loggers. You have specified LOGGER on this PR (I would leave it with the previous name as you changed it to moveit.robot_state ) https://github.com/ros-planning/moveit2/blob/583b6e1368bef879b702636f87ae6c6b8f57eb9a/moveit_core/robot_state/src/robot_state.cpp#L57
and sometimes we use the logger name as *logger_package_name*, lets specify here a proper naming.

Reply:
I believe we settled on `static const  rclcpp:Logger LOGGER` in general. 

For class specific loggers in files with multiple loggers `static const  rclcpp:Logger ROBOT_STATE_LOGGER` would work as well. 

For logger namespacing, we should keep the `"moveit.` prefix. So the logger in RobotState might have a logger:
```c++
static const rclcpp:Logger LOGGER = rclcpp::get_logger("moveit.robot_state")
```
And if there is a reason to use a child logger:
```c++
// within the method
static rclcpp:Logger some_method_logger = LOGGER.get_child("some_method_name")
// outside the method 
static const rclcpp:Logger SOME_METHOD_LOGGER = LOGGER.get_child("some_method_name")
```

Reply:
Okay! thanks for the aclaration, @vmayoral let's note this.

Reply:
@anasarrak that won't work. `static const rclcpp:Logger` will fail. Refer to https://github.com/ros-planning/moveit2/pull/8#discussion_r266211476 for the approach that I've adopted for now.

Reply:
i've updated my [example PR](https://github.com/ros-planning/moveit2/pull/33) to not include ``static``, though it would be nice to better understand why this does not work.

Reply:
I've updated my comment to not include `static`.

Reply:
3181e29691f93558df7735506cf94b613ce0e858

Reply:
This is as agreed at https://github.com/ros-planning/moveit2/issues/21. Resolving this for now (let's dive further into the discussion at https://github.com/ros-planning/moveit2/issues/21 if necessary (e.g. addressing const issues, etc.))